### PR TITLE
Update demo file structure

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #65 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Modernize the structure of how our demo files are created and built
* Update dependencies

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/auro-flightline.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { classMap } from "lit/directives/class-map.js";
 
 import styleCss from "./style-flightline-css.js";
 
+import './auro-flight-segment.js';
+
```
Comment (jason-capsule42) [inline src/auro-flightline.js:13]: Why was this change made?

File: tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["src/**/*.js"], // process the generated JS files from the src js files 
+  "include": ["src/**/*.js"], // process the generated J
```
Comment (jason-capsule42) [inline tsconfig.json:2]: Looks like this change shouldn't have been commited.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/auro-flightline.js (hunk 1 above)
Comment (jordanjones243) [reply src/auro-flightline.js:13]: This change was made because the flight-segment component was not rendering on the example pages

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**